install.sh

```diff
@@ -55,7 +55,7 @@ sudo -H apt-get -y install libssl-dev libcurl4-openssl-dev liblz-dev libbz2-dev
 # Install pip
 ################################################################################
 note_build_stage "Update pip"
-sudo -H apt-get -y install python-dev python3-pip python-wheel
+sudo -H apt-get -y install python-dev python3-pip python-wheel python3-setuptools
```

Comment:
Is this needed to run the action?

Reply:
Yes, because `python3-setuptools` is not installed by default on a vanilla `ubuntu-16.04` instance it seems. This will also be helpful to other users on 16.04 who may need to have that package installed to run Nucleus! 

Without `python3-setuptools` installed, users will get [the following error](https://github.com/suyashkumar/nucleus/runs/408134920?check_suite_focus=true) when running `install.sh`

![Screen Shot 2020-01-27 at 11 39 15 AM](https://user-images.githubusercontent.com/6299853/73207509-aa837e00-40f9-11ea-9bea-dabd5b05c2ce.png)